grype/matcher/common/distro_matchers_test.go

```diff
@@ -53,6 +53,33 @@ func (pr *mockDistroProvider) stub() {
 			},
 		},
 	}
+	pr.data["sles:12.5"] = map[string][]vulnerability.Vulnerability{
+		// direct...
+		"sles": {
+			{
+				Constraint: version.MustGetConstraint("< 2014.1.5-6", version.RpmFormat),
+				ID:         "CVE-2014-fake-4",
+				Namespace:  "sles:12.5",
+			},
+		},
+		// indirect...
+		"sles-ltss": {
```

Comment:
nit: this key isn't being used and could probably be removed. It looks like the above `neutron-devel` case isn't needed either (I think this was added back when there was an intention of doing source-package-indirect searching down in the common distro matcher, but they settled up in each distro matcher instead (deb & rpm)).

Reply:
I've made that change as well, and I removed the existing, unused `neutron-devel` test data.